corona-warn-app/cwa-app-android
# Implement Error Handling for Event Creation (EXPOSUREAPP-5864)

**Testing**

1. Go to `Test Menu > Event Registration > Events > Start Create Event Flow`
2. Create Event
3. Check if the event was created ( in `Test Menu > Show Stored Events`) or an error message was displayed
